test/unit/tools/test_actions.py

```diff
@@ -117,6 +118,14 @@ def test_handler_set(self):
         job, _ = self._simple_execute()
         assert job.handler == TEST_HANDLER_NAME
 
+    def test_inactive_user_job_create_failure(self):
+        self.trans.user_is_active = False
```

Comment:
Isn't a bit misleading that this is something you cannot do in the real `trans`?

No big issue anyway.

Reply:
Yeah and so I was a little wary of doing it. I did find some other places where it's done though.

Is there a better way to do this for unit tests? It all gets a little ugly since there are so many fake objects anyway, which need to be kept at least somewhat in sync with the functionality of the real ones.